addons/binding/org.openhab.binding.mihome/README.md

```diff
@@ -0,0 +1,3 @@
+# Xiaomi Mi Smart Home Binding for OpenHAB 2
+
+TODO
```

Comment:
Please add some simple README file. You can use [the one from my binding](https://github.com/openhab/openhab2-addons/pull/1738/files#diff-00606c14932a3bb70ea25e94ff6d2a5f) as an example